includes/Data/Brands.php

```diff
@@ -6,61 +6,110 @@
+						'utm_medium'	=> reset(explode('/', str_replace(WP_PLUGIN_DIR . '/', '', __DIR__))),
+					),
+				),
+				'fullServiceCreativeTeamInfo' => array(
+					'defaultLink'		=> 'https://www.bluehost.com/solutions/full-service',
+					'fragment'			=> '#full-service',
+					'queryParams'		=> array(
+						'utm_source'	=> 'wp-onboarding',
+						'utm_medium'	=> reset(explode('/', str_replace(WP_PLUGIN_DIR . '/', '', __DIR__))),
+					),
+				),
+				'techSupportInfo' => array(
+					'defaultLink'		=> 'https://helpchat.bluehost.com/',
+					'queryParams'		=> array(
+						'utm_source'	=> 'wp-onboarding',
+						'utm_medium'	=> reset(explode('/', str_replace(WP_PLUGIN_DIR . '/', '', __DIR__))),
+					),
+				),
+			),
+			'bluehost-india' => array(
+				'brand'            => 'bluehost-india',
+				'name'             => 'Bluehost',
+				'url'              => 'https://bluehost.in',
+				'knowledgeBaseUrl' => 'https://www.bluehost.in/hosting/help/results/wordpress',
+				'helpUrl'          => 'https://www.bluehost.in/hosting/help',
+				'blogUrl'          => 'https://www.bluehost.in/tutorials',
+				'facebookUrl'      => 'https://www.facebook.com/BlueHostIndia',
+				'twitterName'      => '@BluehostIndia',
+				'twitterUrl'       => 'https://twitter.com/bluehostindia',
+				'youtubeUrl'       => null,
+				'linkedinUrl'      => null,
+				'accountUrl'       => 'https://my.bluehost.in',
+				'domainsUrl'       => 'https://my.bluehost.in/hosting/app?lil=1#/domains',
+				'emailUrl'         => 'https://my.bluehost.in/hosting/app?lil=1#/email-office',
+				'hireExpertsInfo'   => array(
+					'defaultLink' 		=> 'https://www.bluehost.in/solutions/full-service', 
+					'utmParameters' 	=> array(
 						'utm_source' 	=> 'blog',
 						'utm_medium' 	=> 'socialmedia',
-						'utm_campaign' 	=> 'blogpost')),
-		),
-	);
-
+						'utm_campaign' 	=> 'blogpost'
+					)
+				),
+				'expertsInfo' => array(
+					'defaultLink'		=> 'https://my.bluehost.in/cgi/app/#/marketplace/product/i/bluesky',
+					'queryParams'		=> array(
+						'utm_source'	=> 'wp-onboarding',
+						'utm_medium'	=> reset(explode('/', str_replace(WP_PLUGIN_DIR . '/', '', __DIR__))),
+					),
+				),
+				'fullServiceCreativeTeamInfo' => array(
+					'defaultLink'		=> 'https://www.bluehost.in/solutions/full-service',
+					'fragment'			=> '#full-service',
+					'queryParams'		=> array(
+						'utm_source'	=> 'wp-onboarding',
+						'utm_medium'	=> reset(explode('/', str_replace(WP_PLUGIN_DIR . '/', '', __DIR__))),
+					),
+				),
+				'techSupportInfo' => array(
+					'defaultLink'		=> 'https://helpchat.bluehost.in/',
+					'queryParams'		=> array(
+						'utm_source'	=> 'wp-onboarding',
+						'utm_medium'	=> reset(explode('/', str_replace(WP_PLUGIN_DIR . '/', '', __DIR__))),
+					),
+				),
+			),
+		);
+	}
 	/**
 	 * Get Brand information.
 	 *
 	 * @return array
 	 */
 	public static function get_brands() {
-		return self::$brands;
+		return self::brands_info();
 	}
```

Comment:
Let's remove this extra function and just call `get_brands_info`?

Reply:
removed extra function